Bottom: b8267fd2f582e16abd7766162fc6b1de797d8873
Top:    53eee56debd93fc76921a4f3b27bf2fa732ad390
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 22:18:40 +0100

Refresh of use-trackball-to-allow-rotatio

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index a8e87d5..e43ec87 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -417,7 +417,7 @@ ghid_note_event_location (GdkEventButton * ev)
     }
 
   /* Unproject event_x and event_y to world coordinates of the plane we are on */
-  ghid_unproject_to_z_plane (event_x, event_y, global_depth,
+  ghid_unproject_to_z_plane (event_x, event_y, 0,
                              &event_x, &event_y);
 
   gport->view_x = event_x * gport->zoom + gport->view_x0;
